src/code/relocation.c

```diff
@@ -1,3 +1,123 @@
 #include <global.h>
 
+#ifdef NON_MATCHING
+void Overlay_DoRelocation(void* allocatedVRamAddress, OverlayRelocationSection* overlayInfo, void* vRamAddress) {
```

Comment:
mentioned in discord, would like to see a description of the non matching issues in a comment. other than that this looks good to go